src/include/mpir_handlemem.h

```diff
@@ -55,7 +55,8 @@ int MPIR_check_handles_on_finalize(void *objmem_ptr);
    must ensure that only one thread at a time may call them.
 */
 
-/* This routine is called by finalize when MPI exits */
+/* This routine is called by finalize when MPI exits.  This routine may be called
+ * before initialization and after finalization of MPI. */
```

Comment:
It was my mistake to leave the original comment. I will remove it.

I would like to keep the original comment `this routine may be called before initialization and after finalization of MPI.` since this is an exception in MPICH and the developer must be fully aware of it (e.g., so that the developer may not access any global variables that are initialized during `MPI_Init()`)

Reply:
Sorry, I did not ready the comment below. I will remove all the comments.

Reply:
I added a note at the top of the file instead of adding comments to each function.